### PR TITLE
NoTicket - move jest to dev dependency

### PR DESCRIPTION
**Pull request by lgrzybowski**
I had to move jest to` devDependency` since that package is part of frontend repo now It started to include that jest version and make a clash with our functional tests.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
   },
   "homepage": "https://github.com/BrandwatchLtd/stylelint-config-axiom#readme",
   "devDependencies": {
-    "semantic-release": "^7.0.2"
+    "semantic-release": "^7.0.2",
+    "jest": "^20.0.4"
   },
   "dependencies": {
     "jest": "^20.0.4",
```
Comment (pmsorhaindo) [inline package.json]: This can 💥 in this case then right?

Comment (lgrzybowski) [reply package.json]: true - fixed

**Review by pmsorhaindo — APPROVED** [review]

**Review by stefanbuck — APPROVED** [review]
Looks good to me, but just one question related to this repo.

Is there a particular reason why we want to not include this repo in our axiom monorepo?

**Comment by lgrzybowski** [comment]
I'm not having rights to merge it

**Review by tomru — APPROVED** [review]

**Comment by tomru** [comment]
Just realise this seems to have been moved into the main repo https://github.com/BrandwatchLtd/axiom/tree/master/packages/stylelint-config-axiom

**Pull request merged**